Gemfile

```diff
@@ -44,6 +44,7 @@ group :test, :development do
   gem 'pry-rails'
   gem 'rspec-rails'
   gem 'rack_session_access'
+  gem 'erb2haml'
```

Comment:
Maybe not.. the reason to have it in there is for open PR, what way people could run it on any files they are working on.